lib/commands.js

```diff
@@ -363,14 +374,14 @@ module.exports = (message, command, args) => {
       break
     }
     default: {
-      if (!canExecuteCustomCommand(message)) {
-        return noPermissions(message)
-      }
-
       const value = db.get(command)
 
       if (value) {
```

Comment:
flip this:

if (!value) {
return 
}

if (!canExecuteCustomCommand(message))
{
return
}

etc etc